source/features/dim-bots.tsx

```diff
@@ -4,24 +4,30 @@ import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
-const botSelector = [
+const commitSelector = [
 	/* Commits */
 	'.commit-author[href$="%5Bbot%5D"]',
 	'.commit-author[href$="renovate-bot"]',
-	'.commit-author[href$="scala-steward"]',
+	'.commit-author[href$="scala-steward"]'
+].join();
 
+const prSelector = [
 	/* Issues/PRs */
 	'.opened-by [href*="author%3Aapp%2F"]',
 	'.labels [href$="label%3Abot"]'
 ].join();
```

Comment:
```suggestion
];
```

`select` takes care or the join